### PR TITLE
updated example.md, added example for next app router

### PR DESCRIPTION
**Pull request by s0h311**
> _NOTE_:
>
> - It's a good idea to open an issue first to discuss potential changes.
> - Please make sure that you are _NOT_ opening a PR to fix a potential security vulnerability. Instead, please follow the [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md) to disclose the issue to us confidentially.

## 🧢 Checklist

- [x] Documentation
- [x] Tests
- [x] Ready to be merged

## 🎫 Affected issues

Please [scout and link issues](https://github.com/nextauthjs/next-auth/issues) that might be solved by this PR.

Fixes: INSERT_ISSUE_LINK_HERE

## 📌 Resources

- [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md)
- [Contributing guidelines](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md)
- [Code of conduct](https://github.com/nextauthjs/.github/blob/main/CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

**Review by ThangHuuVu — CHANGES_REQUESTED** [review]
Great attempt, but I think this is a bit trivial to have in the doc 🤔 I would prefer to have one link to the [Next.js doc](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#root-layout-required) and one sentence like:
> Note: SessionProvider requires a client component and cannot be set inside the root layout. For more details, see [the Next.js doc]

How do you think?

**Comment by s0h311** [comment]
> Great attempt, but I think this is a bit trivial to have in the doc 🤔 I would prefer to have one link to the [Next.js doc](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#root-layout-required) and one sentence like:
> 
> > Note: SessionProvider requires a client component and cannot be set inside the root layout. For more details, see [the Next.js doc]
> 
> How do you think?

Sorry for my late response. I like your idea better. But I also think that it should be obvious that this is a way how you can do it with the Next.js App Router. So I propose: 

> Info: If you are using the Next.js App Router plase note that the SessionProvider requires a client component and cannot be set inside the root layout. For more details, see [the Next.js doc]

**Comment by ThangHuuVu** [comment]
SGTM, go for it @s0h311

File: docs/docs/getting-started/example.md [hunk 1]
```diff
@@ -103,6 +103,10 @@ export default function NextAuthProvider() {
 
 Instances of `useSession` will then have access to the session data and status. The `<SessionProvider />` also takes care of keeping the session updated and synced between browser tabs and windows.
 
+:::info
+If you are using the Next.js App Router plase note that the SessionProvider requires a client component and therefore cannot be set inside the root layout. For more details, check out the [Next.js documentation](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts)
```
Comment (ThangHuuVu) [inline docs/docs/getting-started/example.md:78]:
```suggestion
If you are using the Next.js App Router, please note that the `<SessionProvider />` requires a client component and therefore cannot be put inside the root layout. For more details, check out the [Next.js documentation](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts).
```

File: docs/docs/getting-started/example.md [hunk 2]
```diff
@@ -74,6 +74,10 @@ export default function App({
 
 Instances of `useSession` will then have access to the session data and status. The `<SessionProvider />` also takes care of keeping the session updated and synced between browser tabs and windows.
 
+:::info
+If you are using the Next.js App Router, please note that the `<SessionProvider />` requires a client component and therefore cannot be put inside the root layout. For more details, check out the [Next.js documentation](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts).
```
Comment (ThangHuuVu) [inline docs/docs/getting-started/example.md:78]:
```suggestion
If you are using the Next.js App Router, please note that `<SessionProvider />` requires a client component and therefore cannot be put inside the root layout. For more details, check out the [Next.js documentation](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts).
```

File: docs/docs/getting-started/example.md [hunk 3]
```diff
@@ -74,6 +74,10 @@ export default function App({
 
 Instances of `useSession` will then have access to the session data and status. The `<SessionProvider />` also takes care of keeping the session updated and synced between browser tabs and windows.
 
+:::info
+If you are using the Next.js App Router, please note that `<SessionProvider />` requires a client component and therefore cannot be put inside the root layout. For more details, check out the [Next.js documentation](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts).
+:::
+
 :::tip
 Check out the [client documentation](/getting-started/client) to see how you can improve the user experience and page performance by using the NextAuth.js client.
 :::
```
Comment (ThangHuuVu) [inline docs/docs/getting-started/example.md:83]:
```suggestion
:::tip
Check out the [client documentation](/getting-started/client) to see how you can improve the user experience and page performance by using the NextAuth.js client.
If you are using the Next.js App Router, please note that `<SessionProvider />` requires a client component and therefore cannot be put inside the root layout. For more details, check out the [Next.js documentation](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts).
:::
```

**Pull request merged**